fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/RankedSetIndexHelper.java

```diff
@@ -50,6 +54,54 @@
 public class RankedSetIndexHelper {
     public static final Tuple COMPARISON_SKIPPED_SCORE = Tuple.from(Comparisons.COMPARISON_SKIPPED_BINDING);
 
+    public static final RankedSet.HashFunction MURMUR3_HASH_FUNCTION = new GuavaHashFunction(Hashing.murmur3_32());
+
+    /**
+     * Use {@code com.google.common.hash.HashFunction}s as {@code RankedSet.HashFunction}s.
+     */
+    public static class GuavaHashFunction implements RankedSet.HashFunction {
```

Comment:
This kind of feels like a private class, at least until the user can provide their own hash function implementation (instead of using the enum).

Reply:
I moved it out into the replacement for the enum and made it private